language/bert/pytorch_SUT.py

```diff
@@ -15,6 +15,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Modified by SCC23 UCSD Zixian Wang, Nov 11, 2023
```

Comment:
No modification in this file right?

Reply:
There are some places in the code that I modify so that it runs with multiple gpu devices.

Reply:
I think for some reason those changes are not part of this PR. Can you please check?

Reply:
Sorry, I think you are correct. This is not my modified version of pytorch_SUT.py. Should I git push the newer one then start a new pull request?

Reply:
You can just push to this branch and the PR will be updated automatically :)

Reply:
I pushed the modified pytorch_SUT.py. Thank you.